doc/architecture/decisions/0013-p2p-ipc-abstraction.md

```diff
@@ -0,0 +1,32 @@
+# 13. P2P - IPC Abstraction
```

Comment:
pls use 14, I have used 13 for events in #152

Reply:
updated to 14